### PR TITLE
See temperature

### PR DESCRIPTION
**Pull request by eucalvo**

### CODE REVIEW

**Comment by jgfoster** [comment]
Eliam, thanks for merging our latest updates in main to your. That will make it much easier to review this!

File: src/UIState/SeeVersion.cpp [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 void SeeVersion::start() {
   LiquidCrystal_TC::instance()->writeLine(prompt(), 0);
-  const char* version = TankController::instance()->version();
-  LiquidCrystal_TC::instance()->writeLine(version, 1);
+  // const char* version = TankController::instance()->version();
```
Comment (jgfoster) [inline src/UIState/SeeVersion.cpp:12]: Remove commented-out code.

File: test/MenuTest.cpp [hunk 2]
```diff
@@ -62,13 +62,17 @@ unittest(ChangeSettings) {
   assertEqual("Change settings ", lc->getLines().at(0));
   assertEqual("<4   ^2  8v   6>", lc->getLines().at(1));
   enterKey('D');
+  delay(200);  // going back to idle screen requires this
```
Comment (jgfoster) [inline test/MenuTest.cpp:65]: I think that the delay() should not be needed any more.

File: test/PHControlTest.cpp [hunk 3]
```diff
@@ -53,8 +54,9 @@ unittest(bubblerTurnsOnAndOff) {
   assertFalse(controlSolenoid->isOn());
   state->serialPort[1].dataIn = "8.00\r";  // the queue of data waiting to be read
   tc->serialEvent1();                      // fake interrupt to update the current pH reading
-  tc->loop();                              // update the controls based on the current readings
-  assertEqual(13, millis());
+  delay(7);
```
Comment (jgfoster) [inline test/PHControlTest.cpp:57]: Can we try again without the delay()?

**Review by jgfoster — CHANGES_REQUESTED** [review]
Let's talk about the failing tests.

**Comment by eucalvo** [comment]
> 

By failing tests, do you mean the memory space available one?

**Comment by jgfoster** [comment]
@eucalvo, could you contact me at WWU and give me a current email address? Thanks.

**Comment by jgfoster** [comment]
I think this has been superseded by later changes.

**Pull request closed without merging**